### PR TITLE
[STTNHUB-336] Agenda: Show all wire item versions in Coverage preview

### PR DESCRIPTION
**Pull request by MarkLark86**
## Checklist

- [x] This pull request is not adding new forms that use redux
- [x] This pull request is adding missing TypeScript types to modified code segments where it's easy to do so with confidence
- [x] This pull request is replacing `lodash.get` with optional chaining for modified code segments

### CODE REVIEW

**Comment by MarkLark86** [comment]
Desktop:
[STTNHUB-336-desktop.webm](https://github.com/superdesk/newsroom-core/assets/22425232/6b101f4b-08fe-4ab6-b39b-a6f10f233099)

**Comment by MarkLark86** [comment]
Mobile:
[STTNHUB-336-mobile.webm](https://github.com/superdesk/newsroom-core/assets/22425232/c2f6dfb5-7d3f-4072-9341-a914ad9afd27)

**Review by fritzSF — APPROVED** [review]

**Comment by fritzSF** [comment]
Hey @MarkLark86 looks good from the design side.

File: assets/agenda/components/preview/coverage/CoverageActions.tsx [hunk 1]
```diff
@@ -70,61 +91,99 @@ function CoverageActionsComponent({
...
+            .filter((wireId) => wireId.length > 0 && !(hideViewContentItems || []).includes(wireId));
+    const wireVersions = (wireItems || []).filter((item) => wireIds.includes(item._id));
+
+    return wireVersions.length === 0 ? (
+        <div className="coverage-item__row d-flex justify-content-end gap-2">
+            {actionButtons}
+        </div>
+    ) : (
+        <div className={classNames(
+            'nh-collapsible-panel pt-0 nh-collapsible-panel--small coverage-item__extended-row',
```
Comment (tomaskikutis) [inline assets/agenda/components/preview/coverage/CoverageActions.tsx]: It'd be good to extract the UI code for collapsible panel into a separate component. I saw there is `CollapseBoxWithButton` already defined.

**Pull request merged**